.github/workflows/integrationtests.yaml

```diff
@@ -40,7 +37,6 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-
```

Comment:
why?

Reply:
Why what? is this for the dash?  Better to explicitly call it out

Reply:
I mean why this change - it's inconsistent with the spacing between other steps

Reply:
Updated it to keep it consistent.